src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -96,7 +99,27 @@ public async Task<bool> Execute(IPluginCommunication autoRest)
             Configuration.Initialize(autoRest);
 
             var context = new BuildContext(codeModel, null);
+            Execute(context.DefaultNamespace, async (filename, text) =>
+            {
+                await autoRest.WriteFile(filename, text, "source-file-csharp");
+            });
+            return true;
+        }
 
+        public void Execute(string defaultNamespace, string generatedDir)
+        {
+            Execute(defaultNamespace, async (filename, text) =>
```

Comment:
Do we need to update the test cases since we generate csproj file?

Reply:
We need to verify the generated csproj file could work. Could you at least enable one case with --new-project=true?

Reply:
@m-nash The generated csproj cannot pass the build. It references to an invalid build.

Reply:
This is because I am using a private build, in general this won't happen because it will use the exe in the npm package and always have the right version